website/package.json

```diff
@@ -25,15 +25,13 @@
     "@hashicorp/react-search": "^2.1.0",
     "@hashicorp/react-section-header": "2.0.2",
     "@hashicorp/react-subnav": "3.2.6",
-    "@hashicorp/react-text-and-content": "4.1.4",
-    "@hashicorp/react-text-split": "0.4.0",
-    "@hashicorp/react-text-split-with-code": "0.2.0",
-    "@hashicorp/react-text-split-with-image": "1.4.0",
-    "@hashicorp/react-text-split-with-logo-grid": "1.3.0",
+    "@hashicorp/react-text-split": "1.2.0",
+    "@hashicorp/react-text-split-with-code": "2.0.1",
+    "@hashicorp/react-text-split-with-image": "3.0.0",
```

Comment:
`react-text-and-content` is deprecated, and was unused in this repo, so has been removed.
`react-text-split-with-code` was unused in this repo, so has been removed.